src/client/components/atoms/itemList/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Portfolio } from '../../../utils/Type';
+import getAssetUrl from '../../../utils/getAssetUrl';
+import Button from '../button';
+import './style.scss';
+
+type ItemListProps = {
+  item: Portfolio;
+};
+
+const ItemList = ({ item }: ItemListProps) => {
+  console.log('si', item);
+  return (
+    <div className="itemList">

```

Comment:
wrong use of BEM pattern - we don't use camelCase in classNames 

the block should be something like `item-list`